### PR TITLE
RR-1027: Njk, feature flag

### PR DESCRIPTION
**Pull request by lynnette-orr**
https://github.com/user-attachments/assets/e3ac1768-4d39-4ae6-8ff6-4b162a33a4fb

### CODE REVIEW

File: server/views/pages/overview/partials/overviewTab/overviewTabContents.njk [hunk 1]
```diff
@@ -68,6 +68,30 @@
         </div>
       </div>
 
+      {% if featureToggles.sessionsHistoryEnabled %}
```
[the remark this answers is not in the document]
Comment (nathanrussell-moj-digital) [reply server/views/pages/overview/partials/overviewTab/overviewTabContents.njk:71]: I _think_ you want to do something like this in `app.ts`:
Screenshot 2024-11-26 at 08 59 31

And there here in the nunjucks you can do:
```
{% if reviewJourneyEnabledForPrison %}
```

[the remark this answers is not in the document]
Comment (nathanrussell-moj-digital) [reply server/views/pages/overview/partials/overviewTab/overviewTabContents.njk:71]: And then you can revert all your changes above re: setting up a new feature toggle 👍

File: assets/scss/local.scss [hunk 2]
```diff
@@ -49,6 +49,17 @@
...
 }
 
+// Custom tag and container for the sessions data
+
+.govuk-tag.govuk-tag--custom-width {
+   white-space: nowrap; 
+   padding-right: 17px;
+}
+
+.sessions-container {
```
[the remark this answers is not in the document]
Comment (lynnette-orr) [reply assets/scss/local.scss:59]: It's similar to the goals summary card above it. This was the only way I could get the count to line up with the tag.

File: assets/scss/local.scss [hunk 3]
```diff
@@ -49,6 +49,17 @@
   }
 }
 
+// Custom tag and container for the sessions data
+
+.govuk-tag.govuk-tag--custom-width {
```
[the remark this answers is not in the document]
Comment (lynnette-orr) [reply assets/scss/local.scss:54]: The text in the tag was one letter too long to keep it all on one line. Tried a few things but this was the only way I could get it to stay on one line and look right 🤣

**Pull request merged**